src/components/Affidavit/Affidavit.vue

```diff
@@ -452,4 +452,11 @@ ul {
     text-decoration: none;
   }
 }
+
+#upload-affidavit-card {
+    // text area word count counter
+    ::v-deep label,v-label,theme--light {
```

Comment:
This looks like incorrect syntax. I think maybe you mean:
`::v-deep label.v-label.theme--light`

Reply:
"v-label" and "theme-light" are not HTML elements; they are classes, so they need a period in front of them.

This CSS targets all "label"s. Is it working correctly?

Reply:
yeah, it seems to be working properly but only because we probably aren't using any other labels outside of the use case I am looking at.  I'll take another look at this.

Reply:
updated to use correct syntax and tested to be applying styles correctly.